### PR TITLE
feat: migrate certificate-of-awards schema to OpenCerts 2.1, add custom award and signature, update tests and changelog

### PR DESCRIPTION
**Pull request by derrickleemy**
### Summary

This PR updates the `certificate-of-awards/2.0` and `testimonials/2.0` schemas and their associated test suites to align with OpenAttestation v2 conventions and modern best practices. The changes ensure that both schemas and their tests are consistent, robust, and accurately reflect the current requirements.

---

#### **Certificate of Awards v2.0**
- **Schema**
  - Ensured all required fields are defined at the root: `id`, `recipient`, `issuers`, `issuedOn`, `name`, `award`, `signature`.
  - References OpenAttestation v2 schema via `$ref`.
  - Removed obsolete `schema` property.
- **Tests**
  - Refactored to use Ajv with a custom loader for external schema resolution.
  - Removed all tests related to the obsolete `schema` property.
  - Added/updated tests for all root required fields.
  - Added/updated tests for required fields in `award` and `signature`.
  - Ensured the example document is tested for validity.
  - Added tests for additional properties and empty `issuers` array.

#### **Testimonials v2.0**
- **Schema**
  - Ensured all required fields are defined at the root: `id`, `recipient`, `issuers`, `issuedOn`, `name`, `testimonial`, `referee`.
  - References OpenAttestation v2 schema via `$ref`.
  - Removed obsolete `schema` property.
- **Tests**
  - Refactored to use Ajv with a custom loader for external schema resolution.
  - Removed all tests related to the obsolete `schema` property.
  - Added/updated tests for all root required fields.
  - Added/updated tests for required fields in `testimonial`, `referee`, and `recipient`.
  - Ensured the example document is tested for validity.
  - Added tests for additional properties and empty `issuers` array.

---

### Rationale

- **Consistency:** Both schemas and their tests now follow the same structure and validation approach.
- **Modernization:** Tests use Ajv with a custom loader, supporting external schema references and future-proofing validation.
- **Clarity:** Obsolete properties and tests have been removed, reducing confusion and maintenance burden.
- **Robustness:** Test coverage ensures all required fields and edge cases are validated.

---

### Checklist

- [x] All root and nested required fields are tested.
- [x] Example documents are validated.
- [x] Obsolete `schema` property and related tests removed.
- [x] Ajv custom loader used for schema validation.
- [x] Additional properties and edge cases tested.
- [x] Changelogs updated as needed.

---

**This PR brings both schemas and their tests up to date with current OpenAttestation and project standards.**

### CODE REVIEW

File: schema/certificate-of-awards/2.0/schema.json [hunk 1]
```diff
@@ -0,0 +1,584 @@
...
+              "type": "string"
+            }
+          },
+          "required": [
+            "documentStore"
+          ]
+        }
+      ]
+    },
+    "certificateStore": {
```
Comment (skydudie) [inline schema/certificate-of-awards/2.0/schema.json:100]: Should we still keep this since the newer certs would use documentStore?

File: schema/certificate-of-awards/2.0/schema.json [hunk 2]
```diff
@@ -0,0 +1,584 @@
+{
+  "$id": "opencerts/v2.1",
```
Comment (skydudie) [inline schema/certificate-of-awards/2.0/schema.json:2]: I think this should be a unique name to the award

**Comment by skydudie** [comment]
1. Why does it need to inherit all the OpenCerts 2.1 properties? Shouldn't the schema be distinct in its use?
2. I'm not sure if self-contained is the way to go, if we want to ensure that it remains compatible with OA, we might need to maintain that reference to the OA 2.0 schema, otherwise other systems won't be able to tell this is an OA compliant document.

File: schema/testimonials/2.0/schema.json [hunk 3]
```diff
@@ -0,0 +1,601 @@
+{
+  "$id": "opencerts/v2.1",
```
Comment (skydudie) [inline schema/testimonials/2.0/schema.json:2]: Same as above

File: schema/certificate-of-awards/2.0/schema.json [hunk 4]
```diff
@@ -1,584 +1,172 @@
 {
-  "$id": "opencerts/v2.1",
+  "$id": "https://schemas.accredify.io/opencerts/certificate-of-awards/2.0/schema.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "definitions": {
-    "identityProofDns": {
+    "CertificateOfAward": {
```
Comment (skydudie) [inline schema/certificate-of-awards/2.0/schema.json:6]: Is this meant to Pascal Case or Camel Case?

File: schema/testimonials/2.0/schema.json [hunk 5]
```diff
@@ -1,601 +1,211 @@
 {
-  "$id": "opencerts/v2.1",
+  "$id": "https://schemas.accredify.io/opencerts/testimonials/2.0/schema.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "definitions": {
-    "identityProofDns": {
+    "Testimonial": {
```
Comment (skydudie) [inline schema/testimonials/2.0/schema.json:6]: Is this meant to Pascal Case or Camel Case?

Comment (derrickleemy) [reply schema/testimonials/2.0/schema.json:6]: Camel case would be better. Was following their previous implementation

**Review by skydudie — APPROVED** [review]
